pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -216,6 +216,10 @@ func (r *ReconcileIBMBlockCSI) Reconcile(request reconcile.Request) (reconcile.R
 			return reconcile.Result{}, err
 		}
 
+		if err := r.deleteCSIDriver(instance); err != nil {
+			return reconcile.Result{}, err
+		}
```

Comment:
does this cleanup work for all upgrade scenarios on all currently supported k8s versions?

Reply:
yes